runtime/rococo/src/governance/fellowship.rs

```diff
@@ -0,0 +1,358 @@
+// Copyright (C) Parity Technologies (UK) Ltd.
+// This file is part of Polkadot.
+
+// Polkadot is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Polkadot is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Polkadot. If not, see <http://www.gnu.org/licenses/>.
+
+//! Elements of governance concerning the Polkadot Fellowship. This is only a temporary arrangement
+//! since the Polkadot Fellowship belongs under the Polkadot Relay. However, that is not yet in
+//! place, so until then it will need to live here. Once it is in place and there exists a bridge
+//! between Polkadot/Kusama then this code can be removed.
+
+use frame_support::traits::{MapSuccess, TryMapSuccess};
+use sp_arithmetic::traits::CheckedSub;
+use sp_runtime::{
+	morph_types,
+	traits::{ConstU16, Replace, TypedGet},
+};
+
+use super::*;
+use crate::{DAYS, QUID};
+
+parameter_types! {
+	pub const AlarmInterval: BlockNumber = 1;
+	pub const SubmissionDeposit: Balance = 0;
+	pub const UndecidingTimeout: BlockNumber = 7 * DAYS;
+}
+
+pub struct TracksInfo;
+impl pallet_referenda::TracksInfo<Balance, BlockNumber> for TracksInfo {
+	type Id = u16;
+	type RuntimeOrigin = <RuntimeOrigin as frame_support::traits::OriginTrait>::PalletsOrigin;
+	fn tracks() -> &'static [(Self::Id, pallet_referenda::TrackInfo<Balance, BlockNumber>)] {
+		static DATA: [(u16, pallet_referenda::TrackInfo<Balance, BlockNumber>); 10] = [
+			(
+				0u16,
+				pallet_referenda::TrackInfo {
+					name: "candidates",
+					max_deciding: 10,
+					decision_deposit: 100 * QUID,
+					prepare_period: 30 * MINUTES,
+					decision_period: 7 * DAYS,
```

Comment:
not sure if we want the exact same configs, or we want to relax all constraints, to have shorter periods, approve and support curves, etc